File: contains_duplicate.js
// Given an integer array nums, return true if any value appears at least twice in the array, and return false if every element is distinct.


/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(arr) {
    if(new Set(arr).size < arr.length)
    return true;
    return false;
    
};

File: 217-contains-duplicate/contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        
        Set<Integer> set = new HashSet<>();
        for(int x : nums){
            set.add(x);
        }

        if(set.size() < nums.length)
            return true;
        return false;
    }
}